docs-images/package.json

```diff
@@ -0,0 +1,134 @@
+{
+  "name": "docs-images",
+  "displayName": "docs-images",
+  "description": "Docs Images Extension",
+  "icon": "images/docs-logo-ms.png",
+  "version": "0.0.1",
+  "publisher": "docsmsft",
+  "homepage": "https://github.com/Microsoft/vscode-docs-authoring/tree/master/docs-images",
+  "bugs": {
+    "url": "https://github.com/Microsoft/vscode-docs-authoring/issues"
+  },
+  "engines": {
+    "vscode": "^1.22.0"
+  },
+  "repository": {
+    "url": "https://github.com/Microsoft/vscode-docs-authoring.git"
+  },
+  "categories": [
+    "Other"
+  ],
+  "keywords": [
+    "image",
+    "images",
+    "compress",
+    "compression",
+    "optimize",
+    "optimization"
+  ],
+  "activationEvents": [
+    "onCommand:docs.compressImage",
+    "onCommand:docs.compressImagesInFolder"
+  ],
+  "main": "./out/src/extension",
+  "contributes": {
+    "commands": [
+      {
+        "command": "docs.compressImage",
+        "title": "Compress image",
+        "category": "Docs"
+      },
+      {
+        "command": "docs.compressImagesInFolder",
+        "title": "Compress all images in folder",
+        "category": "Docs"
+      }
+    ],
+    "menus": {
+      "commandPalette": [
+        {
+          "command": "docs.compressImage",
+          "when": "false"
+        },
+        {
+          "command": "docs.compressImagesInFolder",
+          "when": "false"
+        }
+      ],
+      "explorer/context": [
+        {
+          "command": "docs.compressImage",
+          "group": "1_modification",
+          "when": "resourceLangId == compressionTargetFile && !listMultipleSelection"
+        },
+        {
+          "command": "docs.compressImagesInFolder",
+          "group": "1_modification",
+          "when": "explorerResourceIsFolder"
+        }
+      ]
+    },
+    "configuration": {
+      "type": "object",
+      "title": "Docs Images Extension Configuration",
+      "properties": {
+        "docsImages.maxWidth": {
+          "type": "number",
+          "default": 1200,
```

Comment:
@adkinn Do you have a preference for the *default* setting for image maximum width in this Docs image compression extension? Most people won't change from the default, so we should pick the ideal Docs maximum as the default.

Reply:
Hi @DuncanmaMSFT,
I based this maximum on the platform suggested max width, called out in the [contributors guide](https://review.docs.microsoft.com/en-us/help/contribute/contribute-how-to-create-screenshot?branch=master#considerations). The max is only used when images are larger than they are recommended to be, and they will maintain the aspect ratio when automatically resized.